backend/editor/entries.py

```diff
@@ -432,20 +432,20 @@ async def get_children(self, entry):
         result = await get_current_transaction().run(query, {"id": entry})
         return await async_list(result)
 
-    async def update_nodes(self, label, entry, new_node_keys):
+    async def update_nodes(self, label, entry, new_node):
         """
         Helper function used for updation of node with given id and label
         """
         # Sanity check keys
-        for key in new_node_keys.keys():
+        for key in new_node.keys():
             if not re.match(r"^\w+$", key) or key == "id":
                 raise ValueError("Invalid key: %s", key)
 
         # Get current node information and deleted keys
         result = await self.get_nodes(label, entry)
         curr_node = result[0]["n"]
         curr_node_keys = list(curr_node.keys())
-        deleted_keys = set(curr_node_keys) ^ set(new_node_keys)
+        deleted_keys = set(curr_node_keys) ^ set(new_node)
```

Comment:
why is it using symmetric difference instead of simple difference here ?

```suggestion
        deleted_keys = set(curr_node_keys) - set(new_node.keys())
```

Reply:
well observed, it should be a simple difference.